packages/server/src/api/controllers/static/templates/preview.hbs

```diff
@@ -89,6 +90,7 @@
       window["##BUDIBASE_HIDDEN_COMPONENT_IDS##"] = hiddenComponentIds
       window["##BUDIBASE_USED_PLUGINS##"] = usedPlugins
       window["##BUDIBASE_LOCATION##"] = location
+      window["##BUDIBASE_SNIPPETS##"] = snippets
```

Comment:
While working inside the builder, snippets are passed across the iframe boundary to ensure that new snippets are immediately available after being updated. Otherwise, we would have to fetch the app definition inside the preview after every change.